includes/radarr/radarr-folder-name-after-year-info.md

```diff
@@ -0,0 +1,3 @@
+<!-- markdownlint-disable MD041-->
+!!! note "Keep in mind adding anything more after the release year could give issues during a fresh import in to Radarr, but it helps for movies that have the same release name and year"
```

Comment:
```suggestion
!!! note "Keep in mind adding anything additional after the release year could give issues during a fresh import into Radarr, but it can help for movies that have the same release name and year"